CHANGELOG.md

```diff
@@ -198,7 +198,7 @@ recommended update.
 
 * Significantly improved manual entry for "qmca" analysis tool, the
   main recommended tool for statistical analysis of QMCPACK data.
-* Added time step fitting tool "qfit" for timestep extrapolation. Uses
+* Added time step fitting tool "qmc-fit" for timestep extrapolation. Uses
```

Comment:
Please do not change the ChangLog it needs to be consistent with the corresponding version

Reply:
We will put a message about renaming in the new release log.